packaging/aznfs/DEBIAN/postinst

```diff
@@ -38,7 +39,25 @@ if [ -f /opt/microsoft/aznfs/randbytes ]; then
         chattr -f +i /opt/microsoft/aznfs/data/randbytes
 fi
 
-# Start aznfswatchdog service.
-systemctl daemon-reload
-systemctl enable aznfswatchdog
-systemctl start aznfswatchdog
\ No newline at end of file
+
+# Check if the config file exists; if not, create it.
+if [ ! -f /opt/microsoft/aznfs/data/config ]; then
+        # Create the config file and set default AUTO_UPDATE_AZNFS=false inside it.
+        echo "AUTO_UPDATE_AZNFS=false" > /opt/microsoft/aznfs/data/config
+
+        # Set the permissions for the config file.
+        chmod 0644 /opt/microsoft/aznfs/data/config
+else
+        chattr -f -i /opt/microsoft/aznfs/data/config
```

Comment:
remove this.. in general don't try to do anything with this file, if present we should just read it and get the config values, else use the default values